test/utils.js

```diff
@@ -1,31 +1,34 @@
 'use strict';
 
-var test = require('tape');
+var expect = require('expect');
 var utils = require('../src/utils');
 
-test('exceptionToString: takes message if present', function(t) {
-  var exception = { message: 'exception message' };
-  var result = utils.exceptionToString(exception);
+describe('utils', function() {
```

Comment:
Besides switching to mocha, how do you feel about making all the acual test files be `*.test.js` or `*.spec.js` ?

Reply:
Done